packages/terra-divider/tests/jest/Divider.test.jsx

```diff
@@ -46,13 +46,13 @@ describe('Divider', () => {
   });
 
   it('should spread a custom prop onto the hr divider', () => {
-    const wrapper = shallow(<Divider id="custom-id" />);
+    const wrapper = shallow(<Divider id="custom-id" text="custom-text" level={3} />);
 
     expect(wrapper.is('#custom-id')).toBe(true);
   });
 
   it('should spread a custom prop onto the text divider', () => {
-    const wrapper = shallow(<Divider id="custom-id" text="custom-text" />);
+    const wrapper = shallow(<Divider id="custom-id" text="custom-text" level={3} />);
```

Comment:
let's not modify the custom props tests. can we add new test for divider with level prop ??
we need similar tests for section-header as well.